yarn.lock

```diff
@@ -14531,16 +14531,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"gatsby-source-rss-feed@npm:1.2.2":
```

Comment:
@skshetry are those changes expected btw? cc @yathomasi

Reply:
I discussed this with @yathomasi, but forgot to make changes to WhatsNewModal. I need to figure out how to keep the modal, but remove this feed link. 

I have reverted this for now: https://github.com/iterative/dvc.org/pull/5370.

Reply:
why removing it though? I think you can redirect it to show latest and RSS from the dvc.org/blog (just in case you missed it). Your call on this.